marketplace.xml

```diff
@@ -698,6 +698,69 @@
     </screenshots>
   </market_entry>
 
+
+  <market_entry>
+    <id>pentaho-jpivot-plugin</id>
+    <type>Platform</type>
+
+    <name>Pentaho Platform JPivot Plugin</name>
+    <description>
+      This is a port of the JPivot functionality that existed in Pentaho Platform 4.8 into a stand alone plugin.
```

Comment:
I'd suggest something like:
«
The legacy and deprecated JPivot plugin.
This plugin is no longer supported. Furthermore, it is vulnerable to known security vulnerabilities.
Hitachi Vantara advises strongly against installing this plugin.
Install this plugin at your own risk.
See the installation instructions, for more information.
»